pkg/metrics/shoot.go

```diff
@@ -294,6 +306,8 @@ func (c gardenMetricsCollector) collectShootNodeMetrics(shoot *gardenv1beta1.Sho
 				worker.Name,
 				worker.Machine.Image.Name,
 				*worker.Machine.Image.Version,
+				criName,
+				strings.Join(containerRuntimes, ", "),
```

Comment:
I'm not so sure about adding a list into a label value. I think it makes more sense to create a new timeseries for each container runtime. @istvanballok wdyt?

Reply:
Based on the PR description:
> `container_runtimes` is the (comma-separated) list of all supported runtimes

then I think it is fine to have the list in a single metric label, because the full list is the one that is supported. However, we should at least make sure that the sorting is consistent.

On the other hand, I'm not sure what the semantics of the supported runtimes really is - maybe it would be sufficient to expose only the used runtime?

Reply:
Based on our call, I sorted the list of container runtimes to keep the order stable.